src/discovery/utils.jl

```diff
@@ -63,6 +63,7 @@ function find_rocm_library(lib::String, dirs::Vector{String}, ext::String = dlex
     isempty(path) || return Libdl.dlpath(path)
 
     for dir in dirs
+        dir = joinpath(dir, "lib")
```

Comment:
On Windows libraries are actually in `bin` subdirectory, not `lib` :/

```suggestion
        dir = joinpath(dir, Sys.islinux() ? "lib" : "bin")
```

Reply:
Lovely but thanks!